be/src/storage/lake/tablet.cpp

```diff
@@ -103,7 +104,13 @@ const std::shared_ptr<const TabletSchema> Tablet::tablet_schema() const {
 }
 
 StatusOr<std::shared_ptr<const TabletSchema>> Tablet::get_schema() {
-    return _mgr->get_tablet_schema(_id, &_version_hint);
+    if (_tablet_schema) {
```

Comment:
when will _table_schema not be nullptr?

Reply:
sorry, here Tablet Construct Function has pushed